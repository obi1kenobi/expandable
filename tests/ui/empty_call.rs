use expandable::expandable;

#[expandable(expr)]
macro_rules! empty {
    () => {}
}

fn main() {}