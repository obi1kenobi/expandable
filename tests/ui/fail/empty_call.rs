// This should fail because "no tokens" is not a valid expression.

#[allow(unused_macros)]
#[expandable::expr]
macro_rules! empty {
    () => {}
}

fn main() {}