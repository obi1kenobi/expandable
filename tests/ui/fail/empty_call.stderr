error: Unexpected end of macro invocation
 --> tests/ui/fail/empty_call.rs:4:1
  |
4 | #[expandable::expr]
  | ^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `expandable::expr` (in Nightly builds, run with -Z macro-backtrace for more info)
